types/interfaces/Asset.ts

```diff
@@ -10,18 +10,18 @@ export declare interface Asset {
   id: string;
   created_at: string;
   status: 'preparing' | 'ready' | 'errored';
-  duration: number;
-  max_stored_resolution: 'Audio-only' | 'SD' | 'HD' | 'FHD' | 'UHD';
-  max_stored_frame_rate: number;
-  aspect_ratio: string;
-  per_title_encode: boolean;
-  is_live: boolean;
-  playback_ids: Array<PlaybackId>;
-  tracks: Array<Track>;
+  duration?: number;
+  max_stored_resolution?: 'Audio-only' | 'SD' | 'HD' | 'FHD' | 'UHD';
+  max_stored_frame_rate?: number;
+  aspect_ratio?: string;
+  per_title_encode?: boolean;
+  is_live?: boolean;
+  playback_ids?: Array<PlaybackId>;
+  tracks?: Array<Track>;
   mp4_support: AssetMp4Support;
-  static_renditions: StaticRenditions;
+  static_renditions?: StaticRenditions;
   master_access: AssetMasterAccess;
   master?: AssetMaster;
-  passthrough: string;
-  errors: AssetError;
```

Comment:
Asset in the errored state is missing a bunch of these keys:

```js
> Video.Assets.get('VmboqxBRX3f53tq3jynQin6xsWcqPEU2').then(console.log)
Promise { <pending> }
> {
  status: 'errored',
  playback_ids: [ { policy: 'public', id: 'UxNkfaRTQ015aoFeh3o684ux2o00n2CA2K' } ],
  mp4_support: 'none',
  master_access: 'none',
  id: 'VmboqxBRX3f53tq3jynQin6xsWcqPEU2',
  errors: {
    type: 'download_failed',
    messages: [
      "The input file 'https://storage.googleapis.com/muxdemofiles/mux-video-int' could not be downloaded."
    ]
  },
  created_at: '1596006613'
}
```

An asset that was created without a playback policy will be missing the `playback_ids` array:

```
> Video.Assets.get('H8Hj3q302XgC3XBAwRjZFoWJsnRgg02uhL').then(console.log)
Promise { <pending> }
> {
  tracks: [
    {
      type: 'video',
      max_width: 1920,
      max_height: 1080,
      max_frame_rate: 29.97,
      id: 'la01NKkJtJQkEnTh8LXy2A9z027K11sUDZ',
      duration: 23.8238
    },
    {
      type: 'audio',
      max_channels: 2,
      max_channel_layout: 'stereo',
      id: 'bR6Kc102A00a8XFBo7wvHerQkDK5wwjT9QINulV101lO1M',
      duration: 23.823792
    }
  ],
  status: 'ready',
  mp4_support: 'none',
  max_stored_resolution: 'HD',
  max_stored_frame_rate: 29.97,
  master_access: 'none',
  id: 'H8Hj3q302XgC3XBAwRjZFoWJsnRgg02uhL',
  duration: 23.857167,
  created_at: '1596120073',
  aspect_ratio: '16:9'
```

Reply:
The only keys that will always be there are:

  * `id`
  * `status`
  *  `created_at`
  * `mp4_support`
  * `master_access`